nmostesting/TestHelper.py

```diff
@@ -264,17 +264,17 @@ def loader(uri):
     return schema
 
 
-def check_content_type(headers, expected_type="application/json"):
+def check_content_type(headers, expected_type="application/json"):
```

Comment:
Not clear in GitHub view what the difference is on this line or others in this file. Please revert all changes to this file.

Reply:
It was an end of line change. We'll make sure to check them in the future.